index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" charset="UTF-8" >
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">
+  </head>
+    <header class="container">
+      <div class="row" id="header">   
+        <nav id="nav-bar" class="navbar navbar-expand-lg navbar-light bg-light">
+         <img 
+         id="header-img"
+         src="/img/logo.jpg"
+         height="100px"
+      >
```

Comment:
You're including the logo in the nav tag, but it should only include elements that include navigation elements. And It's required that we enclose this Logo in an anchor <a> tag. Please consider refactoring it. If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of UX convention make the navigation in the website way more pleasant and intuitive.